src/js/drawer.js

```diff
@@ -50,31 +51,38 @@
           options = $.extend({}, options);
           $this.data(namespace, { options: options });
 
-          var iScroll = new IScroll('.' + __.settings.class.nav, options.iscroll);
+          __.refresh.call(_this);
 
           if (options.showOverlay) {
             __.addOverlay.call(_this);
           }
 
           $('.' + __.settings.class.toggle).on('click.' + namespace, function() {
             __.toggle.call(_this);
-            return iScroll.refresh();
+            return _this.iScroll.refresh();
```

Comment:
Thank you for your PR! This is a minor point, but I prefer it this way. `__.refresh.call(_this);`